src/lib/realmdb/RealmDB.js

```diff
@@ -407,6 +412,27 @@ class RealmDB implements DB, ProfileDB {
   async deleteProfile(): Promise<boolean> {
     return this.profiles.deleteOne({ user_id: this.user.id })
   }
+
+  /**
+   *
+   * @param {TransactionData} data
+   * @returns {Promise<any>}
+   */
+  // eslint-disable-next-line require-await
+  async addToOutbox(data: TransactionData): Promise<any> {
+    return this.inboxes.insertOne({ user_id: this.user.id, ...data })
+  }
+
+  /**
+   *
+   * @param {string} txHash
+   * @returns {Promise<TransactionDetails>}
+   */
+  async getFromOutbox(txHash: string): Promise<TransactionDetails> {
+    const data = await this.inboxes.findOne({ txHash })
+    const decrypted = await this._decrypt(data)
+    return decrypted
```

Comment:
```suggestion

    return decrypted
```

Reply:
Changed in 781edbc9e3778f1c7a05675c3423efe7244504a9